examples/libvirt-caasp-ses.env

```diff
@@ -6,8 +6,8 @@
 # config/*.toml
 
 export ROOKCHECK_HARDWARE_PROVIDER="LIBVIRT"
-export ROOKCHECK_LIBVIRT_CONNECTION="qemu:///system"
-export ROOKCHECK_LIBVIRT_VM_MEMORY="8"
 export ROOKCHECK_DISTRO=SLES_CaaSP
 export ROOKCHECK_NODE_IMAGE_USER=sles
-export ROOKCHECK_LIBVIRT_IMAGE="http://download.suse.de/install/SLE-15-SP1-JeOS-QU4/SLES15-SP1-JeOS.x86_64-15.1-OpenStack-Cloud-QU4.qcow2"
+export ROOKCHECK_LIBVIRT__CONNECTION="qemu:///system"
+export ROOKCHECK_LIBVIRT__VM_MEMORY="8"
+export ROOKCHECK_LIBVIRT__IMAGE="http://download.suse.de/install/SLE-15-SP1-JeOS-QU4/SLES15-SP1-JeOS.x86_64-15.1-OpenStack-Cloud-QU4.qcow2"
```

Comment:
Should be switched to 15SP2 I think

Reply:
Given that we still have CaaSP 4.0, 15SP1 is correct imo